src/app/sequence-editor/sequence-editor.component.ts

```diff
@@ -16,23 +17,35 @@ import { SequenceData } from '../data-models/sequence-model';
   styleUrl: './sequence-editor.component.scss'
 })
 export class SequenceEditorComponent {
-  cardScaleFactor: number = window.innerWidth * (880 / 1280) / 1700;
+  cardScaleFactor: number = Math.min(window.innerWidth * (300 / 1280) / 500, window.innerHeight * (168 / 720) / 282);
   _sequences: SequenceData[] = [];
+  selectedSequence: SequenceData = new SequenceData("error");
+  _flashcards: FlashcardData[] = [];
+  selectedFlashcard: FlashcardData = new FlashcardData();
+
   @Output() addSequenceEvent = new EventEmitter();
   @Output() removeSequenceEvent = new EventEmitter();
-  selectedSequence: SequenceData = new SequenceData("error");
+
   @Input() set sequences(sequence:SequenceData[]) {
     this._sequences = sequence;
     this.selectedSequence = this.sequences[0];
   }
+  @Input() set flashcards(flashcards:FlashcardData[]) {
+    this._flashcards = flashcards;
+    this.selectedFlashcard = this.flashcards[0];
+  }
 
   get sequences() {
     return this._sequences;
   }
 
+  get flashcards() {
+    return this._flashcards;
+  }
+
```

Comment:
Please move the sequence getter right below to the sequence setter. Repeat with the flashcards. Setter/getters should be grouped by the value they are manipulating.